src/main/resources/en.yml

```diff
@@ -317,4 +317,10 @@ random_tp_success: "Random teleport successfully! Be safe."
 custom_gui_not_enough_money: "You don't have enough money to do this!"
 fly_function: 'Fly Command Function'
 fly_enabled: "Now fly is enabled"
-fly_disabled: "Now fly is disabled"
\ No newline at end of file
+fly_disabled: "Now fly is disabled"
+chat_color_nopermission_reply: '§4you without permission "ultikits.tools.chatcolor" so you can not send chat color'
```

Comment:
You do not have permission to send coloured chat messages!

Reply:
ok